test/parallel/test-fs-write-sync-without-offset-and-length.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Ported the changes to test-fs-write-sync-without-offset-and-length.js as well now.

Not sure I'd be comfortable combining them into one (and especially not their async counterparts) -- isn't it better to have more (simple) tests that can fail individually?